core/src/main/java/org/hibernate/ogm/massindex/impl/OgmMassIndexer.java

```diff
@@ -171,23 +173,24 @@ private void atLeastOneValidation(int numberOfThreads) {
 	 * From the set of classes a new set is built containing all indexed
 	 * subclasses, but removing then all subtypes of indexed entities.
 	 *
+	 * @param selection
+	 *
 	 * @return a new set of entities
 	 */
-	private static Set<Class<?>> toRootEntities(ExtendedSearchIntegrator searchFactoryImplementor, Class<?>... selection) {
+	private static IndexedTypeSet toRootEntities(ExtendedSearchIntegrator extendedIntegrator, Class<?>... selection) {
```

Comment:
Just to let you know I noticed this, and yes this is going to be much better with the next round of the Search SPI